Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessBooleanMetadata.java

```diff
@@ -72,15 +72,20 @@ public String getInput() {
 
     @Override
     public Collection<Metadata> getMetadata() throws InvalidMetadataValueException {
+        return getMetadata(true);
```

Comment:
name of method doesn't quite hit it "getMetadataWithoutEmpty" or something else would be better